lib/srv/db/cloud/meta.go

```diff
@@ -99,26 +99,28 @@ func (m *Metadata) fetchRDSMetadata(ctx context.Context, database types.Database
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	// First try to fetch the RDS instance metadata.
-	metadata, err := fetchRDSInstanceMetadata(ctx, rds, database.GetAWS().RDS.InstanceID)
-	if err != nil && !trace.IsNotFound(err) && !trace.IsAccessDenied(err) {
-		return nil, trace.Wrap(err)
+
+	awsMetadata := database.GetAWS()
+
+	if awsMetadata.RDS.ProxyEndpointName != "" {
+		return fetchRDSProxyEndpointMetadata(ctx, rds, awsMetadata.RDS.ProxyEndpointName)
 	}
-	// If RDS instance metadata wasn't found, it may be an Aurora cluster.
-	if metadata == nil {
-		// Aurora cluster ID may be either explicitly specified or parsed
-		// from endpoint in which case it will be in InstanceID field.
-		clusterID := database.GetAWS().RDS.ClusterID
-		if clusterID == "" {
-			clusterID = database.GetAWS().RDS.InstanceID
-		}
```

Comment:
This case seems to be no longer handled. Is it not needed anymore because we would always properly set ClusterID field now when parsing the endpoint?

Reply:
see above comment.